build.gradle

```diff
@@ -78,6 +78,7 @@ ext {
     roomVersion = "2.5.1"
     lifecycleVersion = "2.6.1"
     navigationVersion = "2.5.3"
+    volleyVersion = "1.2.1"
```

Comment:
[Library](https://github.com/google/volley) from Google (Apache 2 Licence) for simple HTTP request - to call `incentives API`.